packages/server/graphql/mutations/createPoll.ts

```diff
@@ -0,0 +1,129 @@
+import {GraphQLNonNull} from 'graphql'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import CreatePollPayload from '../types/CreatePollPayload'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+
+import {GQLContext} from '../graphql'
+import CreatePollInput from '../types/CreatePollInput'
+import MeetingMemberId from 'parabol-client/shared/gqlIds/MeetingMemberId'
+import insertPoll from '../../postgres/queries/insertPoll'
+import insertPollOptions from '../../postgres/queries/insertPollOptions'
+import segmentIo from '../../utils/segmentIo'
+
+const MAX_POLL_OPTIONS = 4
+const MIN_POLL_OPTIONS = 2
+const MAX_TITLE_LENGTH = 100
```

Comment:
min title length? maybe 2?

Reply:
Good idea! I'll move all the consts related to polls to `constEnums` and clean the code.